js/src/Ice/Exception.js

```diff
@@ -38,7 +38,7 @@ const toString = function (key, object, objectTable, ident) {
 //
 // Ice.Exception
 //
-class Exception extends Error {
+export class Exception extends Error {
```

Comment:
We should use/fill Error.cause and Error.message, and definitely not define our own ice_cause property.